### PR TITLE
[FR] Localize Service Discovery

### LINKED ISSUE #2985: [FR] localize Service Discovery
@sestegra
Can i work on 'Localize Service Discovery in French' ?
Best
Seb'

### PR DESCRIPTION
**Pull request by seb-835**
### Describe your changes
Localize Service Discovery

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2985

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/service-discovery.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Les architectures cloud natives sont dynamiques et fluides, ce qui signifie qu'elles changent constamment.
+Une application [conteneurisée](/fr/containerization/) finira probablement par démarrer et s'arrêter plusieurs fois au cours de sa durée de vie.
+À chaque fois que cela se produit, elle aura une nouvelle adresse et
+toute application qui souhaite la trouver aura besoin d'un outil qui lui fournira les nouvelles informations de localisation.
+
+## Quel en est l'utilité
+
+Le mécanisme de découverte de services garde une trace des applications au sein du réseau afin qu'elles puissent se trouver mutuellement en cas de besoin.
+Il fournit une méthode courante pour trouver et potentiellement identifier les services individuels.
+Les moteurs de découverte de services sont des outils semblable à des bases de données qui stockent les informations sur les services existants et comment les localiser.
```
Comment (guillaumebernard84) [inline content/fr/service-discovery.md:22]: Il manque un "s" sur "des outils semblables"

File: content/fr/service-discovery.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Découverte de services (Service Discovery)
+status: Completed
+category: Concept
+tags: ["réseau", "", ""]
+---
+
+La découverte de services est le processus permettant de découvrir les instances individuelles qui composent un service.
```
Comment (guillaumebernard84) [inline content/fr/service-discovery.md:8]: Je pense qu'en français on dirais plutot:
"permettant de découvrir les différentes instances qui composent un service"

Qu'en pensent les autres?

File: content/fr/service-discovery.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Un outil de découverte de service garde une trace des différents nœuds ou points d'extrémité qui constituent un service.
```
Comment (guillaumebernard84) [inline content/fr/service-discovery.md:9]: Il me semble qu'on traduit généralement "endpoint" par "point de terminaison"

File: content/fr/service-discovery.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Les architectures cloud natives sont dynamiques et fluides, ce qui signifie qu'elles changent constamment.
+Une application [conteneurisée](/fr/containerization/) finira probablement par démarrer et s'arrêter plusieurs fois au cours de sa durée de vie.
+À chaque fois que cela se produit, elle aura une nouvelle adresse et
+toute application qui souhaite la trouver aura besoin d'un outil qui lui fournira les nouvelles informations de localisation.
+
+## Quel en est l'utilité
+
+Le mécanisme de découverte de services garde une trace des applications au sein du réseau afin qu'elles puissent se trouver mutuellement en cas de besoin.
+Il fournit une méthode courante pour trouver et potentiellement identifier les services individuels.
```
Comment (guillaumebernard84) [inline content/fr/service-discovery.md:21]: je trouve que "une méthode courante" ne fait pas trés français. 
J'aurais plutot dit "une méthode simple".

Qu'en pensent les autres?

File: content/fr/service-discovery.md (hunk 1 above)
Comment (seb-835) [reply content/fr/service-discovery.md]: fix in https://github.com/cncf/glossary/pull/2998/commits/6edac4e05b2075aace1c8a1166e3e80aacf06996

File: content/fr/service-discovery.md (hunk 3 above)
Comment (seb-835) [reply content/fr/service-discovery.md]: fix in https://github.com/cncf/glossary/pull/2998/commits/6edac4e05b2075aace1c8a1166e3e80aacf06996

File: content/fr/service-discovery.md (hunk 2 above)
Comment (seb-835) [reply content/fr/service-discovery.md]: Fix commit :  https://github.com/cncf/glossary/pull/2998/commits/a7a9943df515b3daa2f0110f448ec703094c056f

File: content/fr/service-discovery.md (hunk 4 above)
Comment (seb-835) [reply content/fr/service-discovery.md]: Fix commit :  https://github.com/cncf/glossary/pull/2998/commits/a7a9943df515b3daa2f0110f448ec703094c056f

**Comment by seb-835** [comment]
@sestegra   @guillaumebernard84  @Krast76 , une petite review quand vous aurez 5 minutes ;)

**Review by guillaumebernard84 — APPROVED** [review]
LGTM 🚀

**Review by sestegra — APPROVED** [review]
LGTM 🚀

**Pull request merged**